### ISSUE TITLE
"put" command not dumped

### ISSUE THREAD

**Issue by aminophen** [aminophen/dviasm#17]
* The 'put' commands (put1, put2, put3, put4) cannot be properly dumped:

```
$ dviasm test/putj.dump.dvi
Traceback (most recent call last):
  File "dviasm.py", line 1252, in <module>
    else:              aDVI.DumpToFile(sys.stdout, tabsize=options.tabsize, encoding=options.encoding)
  File "dviasm.py", line 967, in DumpToFile
    fp.write("put: %s\n" % PutStr(cmd[1]))
  File "dviasm.py", line 237, in PutStrUTF8
    for o in t:
TypeError: 'int' object is not iterable
```

**Comment by aminophen** [comment]
The original ChoF version doesn't seem to work either.  Fixed in 2c5faeb.

I also added a warning message that "only one character is allowed for put!", because `put:` discards the second and subsequent characters.

**Comment by aminophen** [comment]
Released 20200918.
